packaging/systemd/configure-ovs.sh

```diff
@@ -4,8 +4,6 @@
 
 set -eux
 
-nmcli conn show br-ex >/dev/null && echo "br-ex already configured" && exit 0
```

Comment:
Why are we disabling this now?

Reply:
I think the rollback or reconfiguration issue was solved by restarting NM after installing NM-ovs.

Reply:
In the cleanup script, it reverts the init ovs configuration by calling `configure.sh OpenShiftSDN` which doesn't work if we have the br-ex existense check.

Reply:
Ahhh ok I missed that detail